server/aws-lsp-codewhisperer/.eslintrc.js

```diff
@@ -1,9 +1,14 @@
 module.exports = {
     root: true,
     parser: '@typescript-eslint/parser',
-    plugins: ['import'],
+    parserOptions: {
+        projectService: true,
+        tsconfigRootDir: __dirname,
+    },
+    plugins: ['import', '@typescript-eslint'],
     rules: {
         'import/no-nodejs-modules': 'warn',
+        '@typescript-eslint/no-floating-promises': 'error',
```

Comment:
Why not add these rules in top level `.eslintrc.js` for the whole monorepo?

Reply:
Because it will generate much more failures. I started with the project where we develop and the rest (requires more work) needs to be handled as part of different task that I created.